login.js

```diff
@@ -56,8 +56,11 @@ var enterData = function(authData) {
 				answerInput.value = authData['ERPIITKGP_answer3'];
 				sessionStorage.setItem("q3", "1");
 			}
-
-			document.forms[0].submit();
+			const otpdiv = document.getElementById("emailotpdiv");
+			if (otpdiv.className.includes("hidden")) {
```

Comment:
This condition is true momentarily even outside Campus network. It is after the whole page loads that a script runs which removes this condition. Will the extension be able to handle this?